docs/guides/connectingclusters.md

```diff
@@ -0,0 +1,235 @@
+# Connecting TriggerMesh Clusters
+
+!!! Info "Installation"
+    Make sure you have completed the installation [procedure](installation.md) before proceeding with any of the guides.
+
+In this guide we will connect 2 TriggerMesh/Knative clusters that will be able to interchange CloudEvents flowing through them. You might want to connect multiple TriggerMesh instances to:
+
+- Move events between environments. For example from production to staging in order to perform tests with actual events.
+- Geographically distribute events among clusters.
+- Perform Cluster migrations.
+
+## Scenario
+
+For this scenario we will need to setup 2 TriggerMesh clusters that we will call at this document `ClusterSender` and `ClusterReceiver`.
+
+ClusterSender components:
+
+- `Broker` is the temporary storage for CloudEvents.
+- `PingSource` is a peridic CloudEvents producer that will send them to the Broker.
+- `CloudEventsTarget` will listen to CloudEvents and send them to an external location.
+- `Trigger` will subscribe to CloudEvents at the Broker and send them to the CloudEventsTarget.
+
+ClusterReceiver components:
+
+- `Broker` is the temporary storage for CloudEvents.
+- `CloudEventsSource` will listen to CloudEvents from remote locations and send them to the Broker.
+- `event-display` service will listen to CloudEvents and log them at the output console.
+- `Trigger` will subscribe to CloudEvents at the Broker and send them to the EventDisplay.
+
+![connecting clusters](../assets/images/connectingclusters/connectingclusters.png)
+
+Events produced at the PingSource will flow as depicted above until they reach the EventDisplay at the second cluster.
+
+## Setup
+
+!!! Info "Local setup"
+    You can use a local setup by creating 2 kind clusters.
+
+    ```console
+    $ kind create cluster --name clustersender
+    ...
+
+    $ kind create cluster --name clusterreceiver
+    ...
+    ```
+
+    Add LoadBalancer support to the receiver cluster by following [kind instructions](https://kind.sigs.k8s.io/docs/user/loadbalancer/).
+
+
+    You can switch to each configured cluster using `kubectl config use-context` command.
+
+    ```console
+    $ kubectl config use-context kind-clustersender
+
+    $ kubectl config use-context kind-clusterreceiver
+    ```
+
+### Receiver Cluster
+
+!!! Info "Receiver cluster"
+    Make sure your kubectl configuration is pointing to the receiver cluster.
+
+Create the Broker that as the host for this cluster's CloudEvents:
```

Comment:
"that as" => typo?

Reply:
https://github.com/triggermesh/docs/pull/285/commits/979370dec441faf8e38896e11d9d1bfc25eca886